client/gravatar/types.ts

```diff
@@ -0,0 +1,21 @@
+export type GravatarOptions = Partial< {
+	s: number;
+	d: DefaultImage;
+	f?: 'y';
+	forceDefault?: 'y';
+	r?: Rating;
+	rating?: Rating;
+} >;
+
+type DefaultImage =
+	| '404'
+	| 'mp'
+	| 'identicon'
+	| 'monsterid'
+	| 'wavatar'
+	| 'retro'
+	| 'robohash'
+	| 'blank'
+	| URL;
```

Comment:
you may have discovered that this alias of `type URL = string` defeats a big part of the autocomplete on `DefaultImage`. Sad but true because all strings are valid. Regardless, if you start typing one of these or read through the entire autocomplete list you will find them and so it helps some.

Reply:
ah no they do come up in autocomplete for me in vscode 👍 
<img width="626" alt="Screen Shot 2020-02-26 at 7 05 49 PM" src="https://user-images.githubusercontent.com/22446385/75324458-1872bf00-58cb-11ea-9b37-f94cce6e1f4c.png">

Reply:
does it give you an error for `URL`? it may be using the `Node` URL type and instead we need…

```ts
import { URL } from '../types';
```

Reply:
ahh yes that's exactly what was happening